qa-portal-angular/projects/portal-core/src/app/portal-help/portal-help.component.css

```diff
@@ -0,0 +1,33 @@
+.help-bar mat-expansion-panel{
+    overflow: hidden;
+    background-color: #eeeeee;
+    position: fixed;
+    bottom: 0%; 
+    right: 0%;
+    z-index: 20;
+    opacity: 1;
+    width: 20%;
+    white-space: pre-line;
+  }
+
+  .help-bar mat-expansion-panel-header {
+    margin-top: 0px !important;
+  }
+
+  .help-bar mat-panel-title {
+    float: left;
+    display: block;
+    color: #004050;
+    text-align: center;
+    text-decoration: none;
+    padding: 0% 20%;
+    padding-bottom: 5%;
+  }
+
+  .main {
+    margin-bottom: 10%; 
+  }
+
+  .mat-elevation-z2 {
+    box-shadow: 0px 3px 1px -2px rgba(0, 0, 0, 0.2), 0px 2px 2px 0px rgba(0, 0, 0, 0.14), 0px 1px 5px 0px rgba(0, 0, 0, 0.12);
```

Comment:
Why are you overriding the default implementation for this class?

Reply:
Removed override.